File: src/Libro.java
public class Libro {

    private String titulo;
    private String autor;
    private boolean disponible=true;
    public Libro(String titulo,String autor){
        this.titulo=titulo;
        this.autor=autor;
    }
    public String getTitulo() {
        return titulo;
    }
    public String getAutor() {
        return autor;
    }
    public boolean isDisponible() {
        return disponible;
    }
    public void setDisponible(boolean disponible) {
        this.disponible = disponible;
    }

}

File: src/Libro.py
class Libro:
    def __init__(self, titulo, autor):
        self.__titulo = titulo
        self.__autor = autor
        self.__disponible = True

    def get_titulo(self):
        return self.__titulo

    def get_autor(self):
        return self.__autor

    def esta_disponible(self):
        return self.__disponible

    def set_disponible(self, disponible):
        self.__disponible = disponible